meta-phosphor/recipes-phosphor/webui/webui-vue_git.bb

```diff
@@ -6,7 +6,7 @@ LICENSE = "Apache-2.0"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e"
 
 SRC_URI = "git://github.com/ibm-openbmc/webui-vue.git;branch=1030.ips"
-SRCREV = "9f12edba307448a6196f00363cd5459c6ad136ec"
+SRCREV = ""
```

Comment:
Seems the commit id is missing by mistake. @rfrandse will be creating a new bump commit.